templates/components/form/header_content.html.twig

```diff
@@ -128,9 +128,13 @@
    {% if item.isEntityAssign() and is_multi_entities_mode() and item is not instanceof('Entity') %}
       {% set single_actions_ms_auto = false %}
       <span class="badge entity-name mx-1 px-2 ms-auto align-items-center" title="{{ entity_name }}">
-                  <i class="ti ti-stack me-2"></i>
-                  {{ entity_name }}
-               </span>
+        <i class="ti ti-stack me-2"></i>
+           <div style="width: 200px; direction: ltr;">
```

Comment:
IMHO, a fixed width may result in issues on small screens (e.g. mobile phones).

Also, a `span` is an inline HTML tag, it is not supposed to contains blocks HTML tags like a `div`.

Reply:
I made a modification using % instead of px and removed some divs from the span, but I can't figure out how to display only the end of the entity name.